cli/internal/simulation/database/database.go

```diff
@@ -0,0 +1,307 @@
+package database
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"io"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/image"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/docker/api/types/volume"
+	"github.com/docker/docker/client"
+	"github.com/docker/go-connections/nat"
+	"github.com/lib/pq"
+	"github.com/nitrictech/suga/cli/internal/netx"
+	"github.com/nitrictech/suga/cli/internal/style"
+	"github.com/nitrictech/suga/cli/pkg/schema"
+)
+
+const (
+	postgresImage    = "ghcr.io/fboulnois/pg_uuidv7:1.6.0"
```

Comment:
It would be cool to be able to set this as the user or pull it from the plugin (the version at least), so we can test locally with the same postgres version that's being deployed.

Reply:
will make it an env var for now, Ideally it should be a plugin. We need to design an experience where platforms can provide their own information for service emulation.